qp_shotgun/humann2/humann2.py

```diff
@@ -84,50 +85,75 @@ def generate_humann2_analysis_commands(forward_seqs, reverse_seqs, map_file,
             % ', '.join(sn_by_rp.keys()))
```

Comment:
Not sure how to comment on the accordion'd lines, but for filename matching, does this actually work the way we want it to? 

We send IGM sample names, which then get munged and prefixed to some other stuff for the fastq file. So:
[sample1][_Index#_Lane#_Read#].fastq.gz

so `fname` in 64 looks like:
path/to/[sample1][_Index#_Lane#_Read#].fastq.gz

`f` in 65 looks like:
[sample1][_Index#_Lane#_Read#].fastq.gz

`f` in line 68 looks like:
[sample1][_Index#_Lane#_Read#]

this requires that we have all of [_Index#_Lane#_Read#] in our sample map, doesn't it?